django_tasks/__init__.py

```diff
@@ -1,3 +1,8 @@
+# ruff: noqa: E402
+import django_stubs_ext
+
+django_stubs_ext.monkeypatch()
```

Comment:
Suggestion: Could we remove this code from the test's `settings.py`? That way there's only 1 definition, and that way we can know it won't regress.

Reply:
Good point. I removed the django_stubs_ext call from the test's settings.py. Test execution still works as expected.